_data/simple-icons.json

```diff
@@ -2168,6 +2168,11 @@
             "source": "https://www.datadoghq.com/about/resources",
             "guidelines": "https://www.datadoghq.com/about/resources/"
         },
+        {
+            "title": "Dataiku",
+            "hex": "00B2A9",
+            "source": "https://www.dataiku.com"
```

Comment:
Can you update this link to https://www.dataiku.com/company/media-kit/?

Reply:
Done 👍